BWAPI4J/src/main/java/org/openbw/bwapi4j/BW.java

```diff
@@ -240,14 +237,16 @@ private void extractBridgeDependencies(final BridgeType bridgeType) {
     }
   }
 
-  private static void extractFileIfNotExists(
-      final ZipFile zipFile, final String sourceFilename, final String targetDirectory)
-      throws ZipException {
-    final Path targetFile = Paths.get(targetDirectory, sourceFilename);
-    if (!Files.isRegularFile(targetFile)
-        && !Files.isDirectory(targetFile)
-        && !Files.exists(targetFile)) {
-      zipFile.extractFile(sourceFilename, targetDirectory);
+  private static void extractFileIfNotExists(final String sourceFilename, final String targetDirectory) {
+    final File library = new File(targetDirectory, sourceFilename);
```

Comment:
`Path` is more modern and robust than `File`. https://docs.oracle.com/javase/tutorial/essential/io/legacy.html . Please provide justification for using `File` over `Path` in this case.

Reply:
I wasn't aware of the changes but will migrate to using Path if I am able to convert to using `getResourceAsStream`.